plugin/api.go

```diff
@@ -31,6 +31,9 @@ type API interface {
 	// GetConfig fetches the currently persisted config
 	GetConfig() *model.Config
 
+	// GetUnsanitizedConfig fetches the currently persisted config without removing secrets
+	GetUnsanitizedConfig() *model.Config
```

Comment:
lets also include the minimum server version comment as well (similar to other apis)
`// Minimum server version: 5.16`

Reply:
All your suggestions were done @ali-farooq0 :+1: